images/autobump/autobump.sh

```diff
@@ -0,0 +1,195 @@
+#!/bin/bash
+
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+PROW_CONTROLLER_MANAGER_FILE="${PROW_CONTROLLER_MANAGER_FILE:-}"
+# Args for use with GH
+GH_ORG="${GH_ORG:-}"
+GH_REPO="${GH_REPO:-}"
+FORK_GH_REPO="${FORK_GH_REPO:-${GH_REPO}}"
+# Args for use with Gerrit
+GERRIT_HOST_REPO="${GERRIT_HOST_REPO:-}"
+
+# Set this to something more specific if the repo hosts multiple Prow instances.
+# Must be a valid to use as part of a git branch name. (e.g. no spaces)
+PROW_INSTANCE_NAME="${PROW_INSTANCE_NAME:-prow}"
+
+
+
+# TODO(fejta): rewrite this in a better language REAL SOON  <-lol
```

Comment:
?

Maybe we can take the suggestion and rewrite this script in Go and contribute it back to kubernetes/test-infra? :)

Just an idea!

Reply:
Lol who's volunteering =p